src/js/lib/sheettodom.js

```diff
@@ -14,16 +14,18 @@ export function sheetToDomInnerHtml({sheetId, sheetName, el, comingSoonSheetName
             'crossOrigin': true,
             'success': resp => {
                 //get list of elements with data-sheet-attribute
-                for (const node of el.querySelectorAll('[data-sheet-attribute]')) {
-                    const value = node.getAttribute('data-sheet-attribute');
+                Array.from(el.querySelectorAll('[data-sheet-attribute]')).forEach(node => {
+                    const field = node.getAttribute('data-sheet-attribute');
 
-                    if(startsWith(value, `${comingSoonSheetName}-`)){
-                        node.innerHTML = resp.sheets[comingSoonSheetName][0][value.split(`${comingSoonSheetName}-`)[1]];
+                    if(startsWith(field, `${comingSoonSheetName}-`)){
+                        node.innerHTML = resp.sheets[comingSoonSheetName][0][field.split(`${comingSoonSheetName}-`)[1]];
                     }
                     else {
-                        node.innerHTML = resp.sheets[sheetName][0][value];
+                        const docData = resp.sheets.documentaries.find(_ => _.docName === docName);
```

Comment:
I'm going to refactor this in an incoming PR, creating a module for doc metadata and DRYing out the code as we're repeating this here - https://github.com/guardian/docs-interactive-template/pull/134/files#diff-691ff2b74e21c3011d7cec25c48f6146R69